tests/Doctrine/Performance/Mock/NonProxyLoadingEntityManager.php

```diff
@@ -201,9 +201,9 @@ public function detach(object $object): void
         $this->realEntityManager->detach($object);
     }
 
-    public function refresh(object $object): void
+    public function refresh(object $object, LockMode|int|null $lockMode = null): void
     {
-        $this->realEntityManager->refresh($object);
+        $this->realEntityManager->refresh($object, $lockMode);
     }
```

Comment:
Backport to 2.14.x?

Reply:
yes, https://github.com/doctrine/orm/pull/10197